source/fundamentals/authentication.txt

```diff
@@ -142,6 +143,162 @@ string as follow:
+
+      .. code-block:: csharp
+
+         mongoClientSettings.Credential = new MongoCredential("MONGODB-AWS", new MongoExternalIdentity("<awsKeyId>"), new PasswordEvidence("<awsSecretKey>"))
+             .WithMechanismProperty("AWS_SESSION_TOKEN", "<awsSessionToken>");
+
+Retrieve Credentials Automatically
+++++++++++++++++++++++++++++++++++
+
+Instead of specifying your AWS IAM credentials in ``MongoClientSettings``, you can 
+instruct the {+driver-short+} to use the AWS SDK to automatically retrieve your 
+credentials from an external source. To instruct the driver to
+retrieve your credentials, perform the following actions:
+
+- Specify ``MONGODB-AWS`` as the authentication mechanism
+- Specify that the authentication source is external to MongoDB
+- Set your credentials in the appropriate location 
+
+You can specify the authentication mechanism and source either
+by using a ``MongoCredential`` object or as part of the connection string. Select the 
+:guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying the ``MONGODB-AWS`` authentication mechanism
+and external authentication source:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: mongodb-aws-automatic-connection-string
+      
+      .. code-block:: csharp
+
+         var connectionString = "mongodb+srv://<hostname>[:<port>]?authMechanism=MONGODB-AWS&authSource=$external";
+         var mongoClientSettings = MongoClientSettings.FromConnectionString(connectionString);
+         var client = new MongoClient(mongoClientSettings);
+
+   .. tab:: MongoCredential
+      :tabid: mongodb-aws-automatic-mongo-credential
+
+      .. code-block:: csharp
+
+         var mongoClientSettings = MongoClientSettings.FromConnectionString("mongodb+srv://<hostname>[:<port>]");
+         mongoClientSettings.Credential = new MongoCredential("MONGODB-AWS", new MongoExternalAwsIdentity(), new ExternalEvidence());
+         var client = new MongoClient(mongoClientSettings);
+   
+After you specify the authentication mechanism and source, you must set
+your credentials in the location appropriate to the credential type. The {+driver-short+} 
+checks for credentials in the following locations in the order listed here:
+
+- Web identity provider
+- Locally stored credentials
+- Environment variables
+- ECS container credentials
+- EC2 container credentials
+
+You can use an OpenID Connect (OIDC)-compatible **web identity provider** to authenticate
+to Amazon Elastic Kubernetes Service (EKS) or other services.
+To use a web identity provider, create a file that contains your
+OIDC token, then set the absolute path to this file in an environment variable by using 
+``bash`` or a similar shell as shown in the following example:
+
+.. code-block:: bash
+
+   export AWS_WEB_IDENTITY_TOKEN_FILE=<absolute path to file containing your OIDC token>
+
+To authenticate by using **locally stored credentials**, use the AWS CLI to create the 
```

Comment:
This is called `shared aws credentials`. See [here](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/creds-file.html). I would probably rename it and maybe mention that it allows working with profiles (with default profile name: 'default')?